src/cljs/rems/actions/add_licenses.cljs

```diff
@@ -23,7 +23,7 @@
    {:db (assoc db
                ::potential-licenses #{}
                ::selected-licenses #{})
-    :dispatch [:rems.actions.components/set-comment action-form-id nil]
+    :dispatch [:rems.actions.components/set-comment action-form-id ""]
```

Comment:
Do you have plans to also remove the empty string requirement at some point (and change Schema?)

Reply:
I guess the right choice would be to not set the `:comment` field in the command if there's no command.

... but I guess we could also support nil values for the field in the backend.

Reply:
But no, no concrete plans right now